application/src/main/java/uk/nhs/hee/tis/revalidation/service/DoctorsForDBService.java

```diff
@@ -162,19 +161,6 @@ public void updateDoctorConnection(final ConnectionMessageDto message) {
     }
   }
 
-  /**
-   * Disconnect doctors by setting DB to null and existsInGmc to false, then save to repository.
-   *
-   * @param doctorsForDBList list of doctors to be disconnected
-   */
-  public void disconnectDoctorsFromDb(List<DoctorsForDB> doctorsForDBList) {
```

Comment:
What it means to "disconnect a doctor" has changed in the past, and may do in the future (e.g. if we get rid of the existsInGmc flag), is it okay to keep this in its own method?

Reply:
The single use of this method is (currently) quite specific to the context. It relies on conditionally disregarding updates based on `gmcLastUpdatedDateTime`.

It might be more correct to nullify the `existsInGmc` field if & when there is a second use case for this method.  Keen to know if there's something I'm not seeing or might not have considered when inlining this method.